Packs/GoogleCloudFunctions/Integrations/GoogleCloudFunctions/GoogleCloudFunctions.py

```diff
@@ -169,15 +171,37 @@ def format_parameters(parameters: str) -> str:
     return json.dumps(dict(pairs))
 
 
+def set_default_region(region):
+    if region is None:
+        # from Google API : If you want to list functions in all locations, use "-" in place of a location
+        return "-"
+    return region
+
+
+def set_default_project_id(project, credentials_json):
+    if project is None:
+        project = credentials_json["project_id"]
+    return project
+
+
 def main():
     credentials_json = json.loads(demisto.params().get('credentials_json', {}))
-    project = demisto.params().get('project_id', '')
+
+    # when project_id is empty, get it from credentials_json
+    project = demisto.params().get('project_id')
+    project = set_default_project_id(project, credentials_json)
+
+    # when region is empty, set it to '-' meaning all regions
```

Comment:
Place this comment under `set_default_region` function

Reply:
Done